CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ endif()
 
 set( ROCFFT_BUILD_SCOPE ON )
 
+include(GNUInstallDirs)
+
 project( rocfft LANGUAGES CXX C )
 
 # This finds the rocm-cmake project, and installs it if not found
```

Comment:
We need to have only one place that rocm-cmake is downloaded

Reply:
Updated.